tests/acceptance/features/bootstrap/Sharing.php

```diff
@@ -39,22 +39,11 @@ trait Sharing {
 	private int $sharingApiVersion = 1;
 
 	/**
-	 * Contains the API response to the last share that was created by each user
-	 * using the Sharing API. Shares created on the webUI do not have an entry.
+	 * Contains all the user share data
```

Comment:
variable name itself is self-explanatory, so we can remove the comment doc (like with others)